unitylibs/core/workflow/workflow-photoshop/workflow-photoshop.js

```diff
@@ -23,6 +23,21 @@ function addOrUpdateOperation(array, keyToCheck, valueToCheck, keyToUpdate, newV
   }
 }
 
+async function scanForCsam(cfg, id, unityEl, refreshWidgetEvent) {
+  const { scanImgForSafety } = await import('../../steps/upload-step.js');
+  let scanResponse = await scanImgForSafety(cfg, id);
+  if (scanResponse.status === 403) {
+    unityEl.dispatchEvent(new CustomEvent(refreshWidgetEvent));
+    //await showErrorToast(targetEl, unityEl, '.icon-error-acmp');
+    return false;
+  }
```

Comment:
I think we can probably get rid of the `if` logic now, especially if we are not going to show an error anymore for this.